man/gcolor3.1

```diff
@@ -0,0 +1,78 @@
+.TH GCOLOR3 "1" "August 26, 2018"
+.
+.SH NAME
+gcolor3 \- color selection dialog
+.
+.SH SYNOPSIS
+.B gcolor3
+.RI [ OPTIONS ]
+.
+.SH DESCRIPTION
+.B gcolor3
+enables you to pick the color from any pixel on your screen. It also offers a palette, so that you can easily mix and match a couple of colors together.
+.
+.SH OPTIONS
+.SS "Help Options"
+.TP
+.BR \-h ", " \-\^\-help
+Show help options
+.TP
+.BR \-\^\-help\-all
+Show all help options
+.TP
+.BR \-\^\-help\-gtk
+Show GTK+ options
+.SS "GTK+ Options"
+.TP
+.BI "\-\^\-class=" CLASS
+Program class as used by the window manager
+.TP
+.BI "\-\^\-name=" NAME
+Program name as used by the window manager
+.TP
+.BI "\-\^\-gdk\-debug=" FLAGS
+GDK debugging flags to set
+.TP
+.BI "\-\^\-gdk\-no\-debug=" FLAGS
+GDK debugging flags to unset
+.TP
+.BI "\-\^\-gtk\-module=" MODULES
+Load additional GTK+ modules
+.TP
+.BR \-\^\-g\-fatal\-warnings
+Make all warnings fatal
+.TP
+.BI "\-\^\-gtk\-debug=" FLAGS
+GTK+ debugging flags to set
+.TP
+.BI "\-\^\-gtk\-no\-debug=" FLAGS
+GTK+ debugging flags to unset
+.SS "Application Options"
+.TP
+.BR \-v ", " \-\^\-version
+Show the application's version
+.TP
+.BI "\-\^\-display=" DISPLAY
+X display to use
+.
+.SH COPYRIGHT
```

Comment:
Is it common to include the full license in man pages?

Reply:
That's not the full license, it's the recommended subset of the GPL to include in a single file. The full license is MUCH longer.

Full license: https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
Reference for applying: https://www.gnu.org/licenses/gpl-2.0.en.html#SEC4